odata-json-format/10 Media Entity.md

```diff
@@ -37,15 +37,15 @@ A single-valued property or operation response that has the
 [OData-Protocol](#ODataProtocol).
 
 A property or operation response that is of a primitive type is
-represented as an object with a single name/value pair, whose name is
+represented as an object with a name/value pair whose name is
```

Comment:
Deleted "single", because the JSON object can also contain control information and instance annotations.